Submitted upstream: https://gitlab.gnome.org/GNOME/libgweather/-/merge_requests/292

From bfcd8332f45d9e15ad0b8623d5835e06c36dc1f4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
 <congdanhqx@gmail.com>
Date: Tue, 21 Nov 2023 17:51:29 +0700
Subject: [PATCH] Locations: Siem Reap changes its international airport

Old airport won't provide new METAR.
---
 data/Locations.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Locations.xml b/data/Locations.xml
index f4d1e6c6..c42e9508 100644
--- a/data/Locations.xml
+++ b/data/Locations.xml
@@ -3081,7 +3081,7 @@
       </location>
       <location>
         <name>Siem Reap</name>
-        <code>VDSR</code>
+        <code>VDSA</code>
         <coordinates>13.366667 103.850000</coordinates>
       </location>
       <city>
-- 
GitLab

